### PR TITLE
Initial version

### PR DESCRIPTION
**Pull request by dominikholler**
verified by
```
dnf install -y https://resources.ovirt.org/pub/yum-repo/ovirt-release44-pre.rpm
dnf module enable -y javapackages-tools pki-deps postgresql:12
dnf install -y ovirt-engine
engine-setup
# create ovn networks
mkdir repo
cd repo/
dnf install -y createrepo wget
for task in 1333313 1323726 1322380 1322370 ; do   curl -k "https://cbs.centos.org/koji/taskinfo?taskID=$task" | grep "https.*el8.*rpm" | sed -e 's/.*a href="\([^"]*\).*/\1/' |  wget -nc -i -;  done
createrepo $PWD
echo -e "[local]\nname = local\nbaseurl = file://$PWD\nenabled = 1\ngpgcheck = 0\n" > /etc/yum.repos.d/local.repo
dnf update
systemctl status ovn-northd
# check if the network is still there, create other network
```

@maiqueb @sandrobonazzola Can you have a look?

### CODE REVIEW

File: SPEC/ovirt-openvswitch.spec [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+Summary:        Wrapper RPM for upgrading OVS to newer versions
+
+Group:          System Environment/Daemons
+License:        Public Domain
+URL:            http://www.openvswitch.org
+BuildArch:      noarch
+
+Requires:       openvswitch%{ovs_version}
+Provides:       openvswitch = %{ovs_version}
+Obsoletes:      openvswitch <= %{ovs_version_to_replace}
```
Comment (maiqueb) [inline SPEC/ovirt-openvswitch.spec:18]: I'm again puzzled by this; how can you obsolete version 2.11.1 and provide version 2.11 ? To my eyes, you're obsoleting the version you're providing.

Did you ever understood how does this work ?

File: SPEC/ovirt-openvswitch.spec [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+Obsoletes:      ovn <= %{ovs_version_to_replace}
+
+%description ovn
+Wrapper rpm for the base openvswitch-ovn-central package
+
+%package        ovn-central
+Summary:        Wrapper for openvswitch-ovn-central rpm
+License:        Public Domain
+Requires:       ovn%{ovs_version}-central
+Requires:       ovirt-openvswitch-ovn
```
Comment (maiqueb) [inline SPEC/ovirt-openvswitch.spec:59]: who's providing this ?

File: SPEC/ovirt-openvswitch.spec [hunk 3]
```diff
@@ -0,0 +1,153 @@
...
+Obsoletes:      openvswitch-ovn-central <= %{ovs_version_to_replace}
+Obsoletes:      ovn-central <= %{ovs_version_to_replace}
+
+%description ovn-central
+Wrapper rpm for the base openvswitch-ovn-central package
+
+%package        ovn-host
+Summary:        Wrapper for openvswitch-ovn-host rpm
+License:        Public Domain
+Requires:       ovn%{ovs_version}-host
```
Comment (maiqueb) [inline SPEC/ovirt-openvswitch.spec:70]: aren't these the d/s packages ? I'm very confused. Where do they come from if not?

File: SPEC/ovirt-openvswitch.spec (hunk 2 above)
Comment (dominikholler) [reply SPEC/ovirt-openvswitch.spec:59]: The resulting name of the ovn package 15 lines above is ovirt-openvswitch-ovn .

File: SPEC/ovirt-openvswitch.spec (hunk 1 above)
Comment (dominikholler) [reply SPEC/ovirt-openvswitch.spec:18]: You are right, this is odd.
Do you prefer adding epoch or any alternative idea?

File: SPEC/ovirt-openvswitch.spec (hunk 3 above)
Comment (dominikholler) [reply SPEC/ovirt-openvswitch.spec:70]: I want to have them upstream.
You can try via

```
dnf install -y createrepo wget

for task in 1333313 1323726 1322380 1322370 ; do
  curl -k "https://cbs.centos.org/koji/taskinfo?taskID=$task" | grep "https.*el8.*rpm" | sed -e 's/.*a href="\([^"]*\).*/\1/' |  wget -nc -i -
 done

createrepo $PWD
echo -e "[local]\nname = local\nbaseurl = file://$PWD\nenabled = 1\ngpgcheck = 0\n" > /etc/yum.repos.d/local.repo
```

**Review by dominikholler — COMMENTED** [review]
@maiqueb Thanks a lot for your review! I replied to your comments.

**Pull request merged**